src/lib/gas-refund/gas-refund-api.ts

```diff
@@ -168,19 +180,22 @@ export class GasRefundApi {
     endEpoch: number,
   ): Promise<Record<number, boolean>> {
     if (this.network === CHAIN_ID_GOERLI) return {};
-    const claimStatus = await this.merkleRedem.callStatic.claimStatus(
-      address,
-      startEpoch,
-      endEpoch,
-    );
+    const [claimStatusPSP, claimStatusSePSP1] = await Promise.all([
```

Comment:
totally forgot about his piece 🤦 
also - wow, I didn't realise we spend an RPC call for that